NEWS

```diff
@@ -18,6 +18,20 @@ CHANGES WITH 253 in spe:
           For more details, see:
           https://lists.freedesktop.org/archives/systemd-devel/2022-September/048352.html
 
+        * We intend to change behaviour w.r.t. user units and sandboxing
+          options, so that they work in the same way as system units by default.
```

Comment:
"the same way"? not really, they work differently, i.e. uids are all weird.

just say they'll have an effect now, and not gracefully degrade.